python-frontend/src/main/java/org/sonar/plugins/python/api/symbols/ClassSymbol.java

```diff
@@ -33,6 +34,9 @@
 
   Set<Symbol> declaredMembers();
 
+  @Beta
+  Map<Symbol, ClassSymbol> memberTypeHints();
```

Comment:
I think it should return a `Map<Symbol, Symbol>`.
In order to be consistent with AST let's call it `memberTypeAnnotations`